### ISSUE TITLE
Question about the test accuracy

### ISSUE THREAD

**Issue by dk-kim** [weijianan1/NVI#3]
<img width="1176" alt="스크린샷 2024-10-29 오후 2 24 40" src="https://github.com/user-attachments/assets/2e788e03-0d10-4dfb-a8e0-bbd2b19fa8eb">

I set up the environment and ran experiment using the NVI-DEHR/configs/nvi.sh script. 
I modified the data_path argument in the original script from PIC_2.0 to NVI, as downloaded, and changed the pretrained argument from vpic to nvi (since vpic is not available among the pretrained models).

After training, the resulting mean Recall all is 70.68, which is significantly different from the paper’s reported 74.67. 
Could you let me know what might be causing this discrepancy? Any guidance would be appreciated.

**Comment by weijianan1** [comment]
Perhaps the results you have here are from the val set?

**Comment by dk-kim** [comment]
<img width="541" alt="스크린샷 2024-10-30 오후 3 51 54" src="https://github.com/user-attachments/assets/eb17f133-89d7-4f52-b12d-c63c28f7dbdf">

Even after setting according to the newly updated version of the development environment, I got the same result. 
It seems to show results on the validation set using the best validation checkpoint rather than on the test set. 
When I switch to the test set, it reaches 72.46, but this still differs from the performance reported in the paper's table. 
I used configs/nvi.sh for training; please let me know if there's any mistake in the arguments? or could you share the best checkpoint you trained?
